tests/components/alexa/test_smart_home.py

```diff
@@ -217,19 +352,21 @@ def test_api_function_not_implemented(hass):
 
 
 @asyncio.coroutine
-@pytest.mark.parametrize("domain", ['light', 'switch', 'script'])
+@pytest.mark.parametrize("domain", ['alert', 'automation', 'group',
+                                    'input_boolean', 'light', 'script',
+                                    'switch'])
 def test_api_turn_on(hass, domain):
     """Test api turn on process."""
     request = get_new_request(
         'Alexa.PowerController', 'TurnOn', '{}#test'.format(domain))
 
-    # settup test devices
+    # setup test devices
     hass.states.async_set(
         '{}.test'.format(domain), 'off', {
             'friendly_name': "Test {}".format(domain)
         })
 
-    call = async_mock_service(hass, domain, 'turn_on')
+    call = async_mock_service(hass, 'homeassistant', 'turn_on')
```

Comment:
Any reason we would change this to `homeassistant` instead of directly calling the domain? Saves 1 service call -> faster.

Reply:
Oh I guess it doesn't work for groups.

Reply:
Fixed in [this PR](https://github.com/home-assistant/home-assistant/pull/10647) since time is of the essence for Alexa.